src/datamodel_code_generator/model/pydantic/__init__.py

```
@@ -40,9 +40,9 @@ def dict(  # type: ignore[override]
         self, **kwargs: Any
     ) -> dict[str, Any]:
         """Version-compatible dict method for templates."""
-        from datamodel_code_generator.util import PYDANTIC_V2  # noqa: PLC0415
+        from datamodel_code_generator.util import is_pydantic_v2  # noqa: PLC0415
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)